appveyor.yml

```diff
@@ -0,0 +1,50 @@
+# shallow clone
+clone_depth: 10
+
+os: Visual Studio 2015
+
+platform:
+  - x64
+  - Win32
+
+environment:
+  matrix:
+   - Toolset: v140
+   - Toolset: v120
+
+configuration:
+  - Debug
+
+install:
+  - git submodule update --init --recursive
+
+before_build:
+- ps: |
+    md build
+    cd build
+    $generator = switch ($env:TOOLSET)
+    {
+        "v140" {"Visual Studio 14 2015"}
```

Comment:
This modification would be big, need to be test on another testing project, and I will open another pr if it works then